Google/resource-manager/project/vars.tf

```diff
@@ -3,6 +3,12 @@ variable "projects_yml" {
 
 variable "parent_folder" {
   type = string
-  description = "optional: Folder ID of parent folder for set of projects"
+  description = "Numeric ID of parent folder"
```

Comment:
looks like we're trimming the prefix anyway, so maybe ask for it as would be expected on the platform?